spec/workers/hmrc_interface_bulk_submission_worker_spec.rb

```diff
@@ -0,0 +1,107 @@
+require "rails_helper"
+require 'sidekiq/testing' # Warning: Requiring sidekiq/testing will automatically call Sidekiq::Testing.fake!, see https://github.com/sidekiq/sidekiq/wiki/Testing
+
+RSpec.describe HmrcInterfaceBulkSubmissionWorker, type: :worker do
+  describe ".perform_async" do
+    subject(:perform_async) { described_class.perform_async }
+
+    it "enqueues 1 job with expected options" do
+      expect { perform_async }
+        .to change(described_class, :jobs)
+          .from([])
+          .to(
+              [
+                hash_including(
+                  "retry" => true,
+                  "queue" => "default",
+                  "args" => [],
+                  "class" => "HmrcInterfaceBulkSubmissionWorker"
+                )
+              ]
+          )
+    end
+
+    # TODO: while this works it redifines the class rendering described_class unusable
```

Comment:
as with other similar tests in other spec it would be good to test this UAT logic but have not found a solid way of doing it